pkg/controller/controller.go

```diff
@@ -218,20 +218,26 @@ func (c *Controller) onUpdateActionSet(oldAS, newAS *crv1alpha1.ActionSet) error
 		log.Print("Updated ActionSet", field.M{"ActionSetName": newAS.Name})
 		return err
 	}
+	ctx := context.TODO()
+	ctx = field.Context(ctx, consts.ActionsetNameKey, newAS.GetName())
```

Comment:
The action set name is useful in the function logs below but looks like all the logs in the controller already have it as a field. How do we avoid duplication? Should we update the logs to not explicitly log the name?

Reply:
@pavannd1 I will update functions to not use ActionName in log.